libsrc/c64/soft80_cgetc.s

```diff
@@ -65,20 +75,7 @@ invertcursor:
         ; do not use soft80_putcolor here to make sure the cursor is always
         ; shown using the current textcolor without disturbing the "color voodoo"
         ; in soft80_cputc
```

Comment:
That comment block belongs to the code that was moved to an inlined position.  It, too, should be moved.